docs/python/environments.md

```diff
@@ -117,29 +77,19 @@ python3 -m venv .venv
 python -m venv .venv
 ```
 
-When you create a new virtual environment, a prompt will be displayed to allow you to select it for the workspace.
+When you create a new virtual environment, a prompt will be displayed to allow you to select it for the workspace. To learn more about the `venv` module, see [Creation of virtual environments](https://docs.python.org/3/library/venv.html) on Python.org.
```

Comment:
Could we swap these two sentences? i.e. move the "to learn more about the module" sentence to the beginning and then say that when you create a new venv the prompt will be displayed? they could be even be in separate paragraphs, I think 🤔

Reply:
Swapped the order and made the "To learn more" a note.